CHANGELOG.asciidoc

```diff
@@ -47,6 +47,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 *Metricbeat*
 
 - Change field type of http header from nested to object {pull}5258[5258]
+- Use `beat.name` instead of `beat.hostname` in the Host Overview dashboard. {pull}5340[5340]
+- Auto-select a hostname (based on the host on which the Beat is running) in the Host Overview dashboard. {pull}5340[5340]
```

Comment:
That is more a feature then a bug fix?

Reply:
Depends how you consider it. It's fixing having an empty dashboard if no host is selected.

Reply:
I'll move it to Added, I think it makes more sense there.